### PR TITLE
Add tooltip to format tool bar

### PR DESCRIPTION
**Pull request by Jocs**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| Fixed tickets    | #1515 
| License          | MIT

### Description

[Description of the bug or feature]

--

#### Please, don't submit `/dist` files with your PR!

---
This change is [Reviewable](https://reviewable.io/reviews/marktext/marktext/1516)

### CODE REVIEW

File: src/muya/lib/ui/formatPicker/config.js [hunk 1]
```diff
@@ -11,30 +11,39 @@ import clearIcon from '../../assets/pngicon/format_clear/2.png'
...
     type: 'inline_code',
+    tooltip: 'Inline Code',
     icon: codeIcon
   }, {
     type: 'inline_math',
+    tooltip: 'Inline Math',
     icon: mathIcon
   }, {
     type: 'link',
+    tooltip: 'Line',
```
Comment (fxha) [inline src/muya/lib/ui/formatPicker/config.js]: Typo `Link`

**Comment by fxha** [comment]
mt_overlay_hover

@Jocs The tool-tip is not fully visible on first line.

**Comment by Jocs** [comment]
>The tool-tip is not fully visible on first line.

I didn't find a better to solve this bug only by css. maybe we can solve it latter, do you have any idea?

**Comment by fxha** [comment]
@Jocs I like the old design/text more than the current one. I don't really know what to do with this information as a user (ok it's a shortcut in macOS style but IDK).

**Comment by Jocs** [comment]
> I like the old design/text more than the current one.

Do you think it's no need to add tooltip to format tool bar?

**Comment by fxha** [comment]
I think it's a good idea but it's should be text. Showing a shortcut only without information is a not the best way to go. What about the HTML5 `title` attribute that is positioned by browser?

**Comment by Jocs** [comment]
>Showing a shortcut only without information is a not the best way to go.

I think tooltip with shortcut is necessary, some people is will not always remember the shortcut of of each menu item, `title` attribute with delay and without good UI.

**Comment by fxha** [comment]
Why not just show `<title> <shortcut>`  if you want shortcuts too. We also have the command palette soon for better navigation. Could get or calculate the window offset of the format overlay to show the tooltip at the bottom?

**Comment by Jocs** [comment]
>Why not just show <title> <shortcut> if you want shortcuts too. 

Do you mean put `description + shortcut` in the title attribute of element? 

>Could get or calculate the window offset of the format overlay to show the tooltip at the bottom?

Some complicated, I feel that for the tooltip, to do the work, it is superfluous.

**Comment by fxha** [comment]
> Do you mean put `description + shortcut` in the title attribute of element?

Yes exactly.

**Comment by Jocs** [comment]
@fxha Add title attribute and remove the tooltip.

**Pull request merged**